app/src/main/java/org/mozilla/focus/menu/home/HomeMenu.kt

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.menu.home
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.support.v7.widget.LinearLayoutManager
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.PopupWindow
+import kotlinx.android.synthetic.main.menu.view.*
+import org.mozilla.focus.R
+import org.mozilla.focus.ext.dp
+import org.mozilla.focus.utils.ViewUtils
+
+class HomeMenu(
+        val context: Context,
+        val listener: View.OnClickListener
+) : PopupWindow(), View.OnClickListener {
+    init {
+        contentView = LayoutInflater.from(context).inflate(R.layout.menu, null)
+
+        with(contentView.list) {
+            layoutManager = LinearLayoutManager(context, LinearLayoutManager.VERTICAL, false)
+            adapter = HomeMenuAdapter(context, this@HomeMenu)
+            setPadding(0, context.dp(8), 0, 0)
```

Comment:
You should be passing in pixel values to `setPadding`, not dp, right?

Reply:
Yeah, correct. The method converts dp to pixels. Maybe I should rename it? I thought it would be more readable this way: "I want the padding to be 8dp". Anyhow, just talked to antlam yesterday and we want to have the padding consistent in all menus. So I can move this from here to the layout file. :)